pkg/webui/console/views/application-overview/index.js

```diff
@@ -40,39 +41,41 @@ class ApplicationOverview extends React.Component {
       updated_at,
     } = this.props.application
 
+    const sheetData = [
+      {
+        header: sharedMessages.generalInformation,
+        items: [
+          { key: sharedMessages.appId, value: ids.application_id, type: 'code', sensitive: false },
+          { key: sharedMessages.joinEUI, value: '00000000000000', type: 'byte', sensitive: false },
```

Comment:
`00000000000000` ?

Reply:
Removed. We don't have that info in the entity.